### PR TITLE
chore: bump deps and major version

### PR DESCRIPTION
**Pull request by ziimakc**
- update deps
- ~update major version~
- update `peerDependencies` `mercurius` to 11 (fixes pnpm warnings)
- set min node to 14 as required by husky and mercurius

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mercurius-integration-testing",
-  "version": "5.0.0",
+  "version": "6.0.0",
```
Comment (mcollina) [inline package.json]: please revert this, we'll bump when we release

**Review by mcollina — CHANGES_REQUESTED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by ziimakc** [comment]
@PabloSzx can we merge?

**Review by PabloSzx — APPROVED** [review]

**Comment by PabloSzx** [comment]
seems good, thanks for @ me

**Pull request merged**